src/smc-hub/postgres-base.coffee

```diff
@@ -315,6 +319,13 @@ class exports.PostgreSQL extends EventEmitter    # emits a 'connect' event whene
                         clearTimeout(timeout)
                         cb(err)
                 async.map(locals.clients, f, cb)
+            (cb) =>
+                # we set a statement_timeout, to avoid queries locking up PG
+                f = (client, cb) =>
+                    sto = DEFAULT_STATEMENT_TIMEOUT_s * 1000 # in millisecs
```

Comment:
Can you please make the variable name clear, e.g., `statement_timeout_ms` instead of `sto`?